domenetjenester/mottak/src/main/java/no/nav/foreldrepenger/mottak/dokumentpersiterer/impl/søknad/v3/TilretteleggingOversetter.java

```diff
@@ -148,24 +149,24 @@ private SvpTilretteleggingEntitet oppdaterEksisterendeTlrMedNyeFomsOgOpphold(Svp
 
         nyFomListe.sort(Comparator.comparing(TilretteleggingFOM::getFomDato));
 
-        var justerteOppholdsperioder = eksisterendeTlr.getAvklarteOpphold().stream()
-            .filter(opphold -> opphold.getFom().isBefore(tidligsteNyFom))
-            .map(opphold -> {
-                if (!opphold.getTom().isBefore(tidligsteNyFom)) {
-                    var eksisterendeKilde = opphold.getKilde() != null ? opphold.getKilde() : SvpOppholdKilde.REGISTRERT_AV_SAKSBEHANDLER;
-                    return SvpAvklartOpphold.Builder.nytt()
-                        .medKilde(eksisterendeKilde)
-                        .medOppholdÅrsak(opphold.getOppholdÅrsak())
-                        .medOppholdPeriode(opphold.getFom(), tidligsteNyFom.minusDays(1))
-                        .build();
-                }
-                return opphold;
-            }).toList();
+        //Vi beholder innvilgede oppholdsperioder som er før nyeste svangerskapspengeperiode
+        var eksisterendeOppholdSomSkalBeholdes = eksisterendeTlr.getAvklarteOpphold().stream()
+            .filter(eksOpphold -> eksOpphold.getFom().isBefore(tidligsteNyFom))
+            .map(eksOpphold ->
+                    SvpAvklartOpphold.Builder.nytt()
+                        .medKilde(SvpOppholdKilde.TIDLIGERE_VEDTAK)
```

Comment:
Hm, vil du da ikke kunne få få 2 sett med like perioder - en med tidligere vedtak og en med søknad?
Eksempel: Sak fra TFP, ferien er lik i de to søknadene, men slår litt ut med tidligsteFom.
Ganske usikker på dette området :-)